src/controllers/VariableController.ts

```diff
@@ -100,6 +100,20 @@ export class VariableController {
         return res.setVariableType(variableId, type).then((result) => getEditorResponseData<null>(result));
     };
 
+    /**
+     * This method sets the items of the list variable
+     *
+     * The items need to be unique and are not case sensitive.
+     *
+     * @param id the id of the list variable
+     * @param items the items of the list
+     * @returns
+     */
+    setListVariable = async (id: string, items: string[]) => {
```

Comment:
id: Id (sorry, forgot about this one!) If items are also ids, then items: Id[].

Reply:
**Oops** indeed, however the items are not `Id`'s.